tests/Unit/NumericalAlgorithms/LinearOperators/Python/Test_PowerMonitors.py

```diff
@@ -0,0 +1,41 @@
+#Distributed under the MIT License.
+#See LICENSE.txt for details.
+
+from spectre.LinearOperators import power_monitor_array
+from spectre.Spectral import (Mesh2D, Basis, Quadrature)
+from spectre.DataStructures import DataVector
+
+import numpy as np
+import unittest
+
+
+class TestPowerMonitors(unittest.TestCase):
+    # Check the case for a constant function where the power monitors
+    # should be given by the first basis function
+    def test_power_monitor_array(self):
+        elements_per_dimension = 4
```

Comment:
Name this maybe `num_points`. The word "element" means a finite-element cell in our code, which has a grid of points.